srcpkgs/66/template

```diff
@@ -0,0 +1,38 @@
+# Template file for '66'
+pkgname=66
+version=0.1.0.0
+revision=1
+wrksrc="66-v${version}"
+build_style=gnu-configure
+configure_args="--with-sysdeps=${XBPS_CROSS_BASE}/usr/lib/skalibs/sysdeps
+ --with-lib=${XBPS_CROSS_BASE}/usr/lib/oblibs
+ --with-lib=${XBPS_CROSS_BASE}/usr/lib
+ --with-dynlib=${XBPS_CROSS_BASE}/usr/lib
+ --with-include=${XBPS_CROSS_BASE}/usr/lib/include"
+makedepends="oblibs-devel skalibs-devel execline-devel s6-devel s6-rc s6-portable-utils"
+depends="s6-rc s6-portable-utils"
+short_desc="Helpers tools around s6-rc"
+maintainer="mobinmob <mobinmob@disroot.org>"
+license="ISC"
+homepage="http://web.obarun.org/software/"
+distfiles="https://framagit.org/Obarun/66/-/archive/v${version}/66-v${version}.tar.bz2"
+checksum=dc6a84a4f0aca6b43da98d74bc5162794061a41acf831bdf50b771ce523b0996
+
+post_install() {
+	local i
+	vlicense COPYING
+	for i in doc/*.html
```

Comment:
use `; do` instead of putting the `do` in a new line.

Reply:
Done :)